spec/features/residents_can_see_confirmation_of_repair_request_spec.rb

```diff
@@ -273,6 +277,10 @@
       priority: 'N',
       problemDescription: "My sink is blocked\n\nRoom: Other\n\nCallback requested: between 8am and 5pm",
       propertyReference: '00000503',
+      contact: {
+        name: 'John Evans',
+        telephoneNumber: '078 98765 432',
+      },
```

Comment:
Does this data come back from the `get_repair` requests? If so should we update our stubs (above) to reflect this?

Reply:
Possibly. We're not currently using the retrieved data (from `get_repair`) on the confirmation page, but maybe should do. The data in the session and the data we get back from UH should be identical anyway.

Maybe one for another card which uses more of the data returned from the API rather than parroting back the user input(?)